src/main/java/reposense/authorship/BlameAnalyzer.java

```diff
@@ -0,0 +1,38 @@
+package reposense.authorship;
+
+import java.util.Map;
+
+import reposense.dataobject.Author;
+import reposense.dataobject.RepoConfiguration;
+import reposense.system.CommandRunner;
+
+/**
+ * Analyzes the authorship of a {@code FileInfo} using the git blame result.
+ */
+public class BlameAnalyzer {
+    private static final int AUTHOR_NAME_OFFSET = "author ".length();
+
+    /**
+     * Sets the {@code Author} for each line in {@code fileInfo} based on the git blame analysis on the file.
+     */
+    public static void aggregateBlameAuthorInfo(RepoConfiguration config, FileInfo fileInfo) {
```

Comment:
since this class and method is only used in `FileInfoAnalyzer` as main functionality, should we move this to `FileInfoAnalyzer` instead?

Reply:
Hmmm, in this case, should `AnnotatorAnalyzer` and `CheckStyleParser` be inside `FileInfoAnalyzer` since they are also part of the file analysis functionality as well?

Reply:
they're sort of sub-plugin... perhaps make them sub packages?